### PR TITLE
Add custom-made spinner replacement .gif

### LINKED ISSUE #1263: [Feature Request] New Loading GIF
Pedantic request here but would it be possible to update the 'Loading...' gif that appears prior to rendering a flexdashboard? It feels a bit outdated and pixelated compared to the 'Please wait...' version that appears on regular shiny apps.

Initially filed this in the flexdashboard repo but was advised to file here instead.

Thanks!

### LINKED ISSUE #1328: R Markdown Websites no longer work with non `html_document` formats
R Markdown Websites were originally designed to work with any HTML based format (so long as it supports the site_libs and self_contained options). However, a recent change intended to address this issue (https://github.com/rstudio/rmarkdown/issues/793) has broken this:

https://github.com/rstudio/rmarkdown/pull/1177

So now the feature which originally supported any HTML based format can only work with html_document.

The original issue #793 is certainly also a legitimate use case (rendering PDFs alongside HTML documents) but this particular fix is too drastic, so I think should be reverted in preference to another fix (PR to follow with the revert, I'm not at all sure what the alternate fix should be, perhaps instantiating the output format and checking it's formals before forwarding on those parameters?).

### PR DESCRIPTION
**Pull request by rich-iannone**
Fix #1263 I created a custom spinner gif (using the RStudio blue from the new orb). It's not a pure CSS solution but it does look decent as a drop-in replacement and has a relatively small file size.

### CODE REVIEW

**Comment by yihui** [comment]
Here is the spinner used on ShinyApps.io:

spinner

**Comment by jjallaire** [comment]
I agree we should use the same one as ShinyApps.io if possible....

**Comment by rich-iannone** [comment]
New loading .gif is in place. I tested it with an example flexdashboard .Rmd and it looks great.

**Comment by rich-iannone** [comment]
@yihui That would involve remaking the .gif. This is something I could totally do, can't guarantee it will look exactly the same but it should be very close. I'll also try to get it to be low in byte size.

**Comment by yihui** [comment]
I meant `tags$p("Please wait...")` (instead of the text in the image).

**Comment by rich-iannone** [comment]
Oh yes, of course! I will do that.

File: R/shiny.R [hunk 1]
```diff
@@ -373,7 +373,7 @@ rmarkdown_shiny_ui <- function(dir, file) {
           id = "rmd_loader_wrapper",
           tags$div(id = "rmd_loader", style = "display: none",
                    tags$img(src = "rmd_resources/rmd_loader.gif"),
-                   tags$p("Loading")))),
+                   tags$p("Please wait...")))),
```
Comment (yihui) [inline R/shiny.R]: Please move the text above the gif (that is what ShinyApps.io shows on the loading page).

**Comment by rich-iannone** [comment]
You're welcome. BTW, I'm learning a lot!

**Pull request merged**